docs/docs/reference/02-manual-installation-launch.md

```diff
@@ -0,0 +1,311 @@
+image. This is done automatically when building with newer versions of `app-builder`. This means you won't be able to
+use older versions of certain libraries and packages with newer versions of `app-builder`.
+
+:::
+
+```toml title="aica-application.toml"
+#syntax=ghcr.io/aica-technology/app-builder:v2
+
+[core]
+"image" = "v4.2.0"
+
+[packages]
+# add components
+"@aica/components/rl-policy-components" = "v2.0.0"
+
+# add hardware collections
+"@aica/collections/ur-collection" = "v4.1.0"
+```
+
+### Including custom packages
+
+The AICA framework allows developers to build their own
+[custom components](../reference/custom-components/01-component-package.md). These packages can be included under a
+custom name using the `docker-image://` prefix to specify the docker image name or path. For example, a custom component
+package that was locally built using `docker build [...] --tag my-custom-component-package` could be included as
+`docker-image://my-custom-component-package`. Community and third-party packages may also be available on other docker
+registries such as DockerHub or GitHub Container Registry and can be included with the associated docker path.
+
+```toml title="aica-application.toml"
+#syntax=ghcr.io/aica-technology/app-builder:v2
+
+[core]
+"image" = "v4.2.0"
+
+[packages]
+# add a custom package from a local docker image path
+"my-local-package" = "docker-image://my-custom-component-package"
+
+# add a package from any docker path such as GitHub Container Registry
+"my-ghcr-package" = "docker-image://ghcr.io/user/package:tag"
+```
+
+## Building an AICA runtime application image
+
+:::note
+
+[Log in to the package registry](#logging-in-to-the-aica-package-registry) before building the image to authorize docker
+to access AICA packages.
+
+:::
+
+Once the desired packages have been configured in a manifest file, a `docker build` command can be used to build the
+runtime application image. In this example, a manifest file saved as `aica-application.toml` is used to build an image
+with the name `aica-runtime`.
+
+```shell
+docker build -f aica-application.toml -t aica-runtime .
+```
+
+The command `docker image ls | grep aica-runtime` should then list the `aica-runtime` image.
+
+
+## Starting the application container
+
+You can start the AICA application container with the following command.
+
+:::note
+
+Change `/path/to/license` in the command below to the location of the `aica-license.toml` file from above. For example,
+use `~/.aica-license.toml` to keep the license file hidden in the home folder.
+
+:::
+
+<!-- TODO: edit this if we ever go away from --net=host -->
+
+<Tabs groupId="os">
+<TabItem value="linux" label="Linux">
+
+```bash
+docker run -it --rm \
+  --privileged \
+  --net=host \
+  -v /path/to/license:/license:ro \
+  aica-runtime
+```
+
+</TabItem>
+<TabItem value="mac" label="macOS">
+
+```bash
+docker run -it --rm \
+  --privileged \
+  -p 8080:8080 -p 18000-18100:18000-18100/udp \
```

Comment:
Is it worth noting that you can remap `8080` to something else but not `18000-18100`?

Reply:
Added a note